addon/mixins/route.js

```diff
@@ -132,18 +133,22 @@ const RouteMixin = Mixin.create({
 
     if (options.store) {
       if (options.storeFindMethod) {
-        this.set('_storeFindMethod', options.storeFindMethod);
+        set(this, '_storeFindMethod', options.storeFindMethod);
       }
 
       this._ensureCustomStoreCompatibility(options);
 
-      this.set('_store', options.store);
+      set(this, '_store', options.store);
 
       delete options.store;
       delete options.storeFindMethod;
     }
 
-    const currentPage = options.startingPage === undefined ? 0 : options.startingPage-1;
+    // default is to start at 0, request next page and increment
+    const currentPage = options.startingPage === undefined ? 0 : options.startingPage - 1;
+    // sets first page when route is loaded
+    const firstPage = currentPage === 0 ? 1 : currentPage + 1;
```

Comment:
wanted to call this `startingPage`, but we already have a property to pass to `infinityModel` called startingPage.